GCRCatalogs/cosmodc2.py

```diff
@@ -286,7 +286,8 @@ def _iter_native_dataset(self, native_filters=None):
             with h5py.File(file_path, 'r') as fh:
                 for group in self._get_group_names(fh):
                     # pylint: disable=E1101,W0640
-                    yield lambda native_quantity: fh['{}/{}'.format(group, native_quantity)].value
+                    if len(fh[group]) > 0:
```

Comment:
minor code style change:
```suggestion
                    if len(fh[group]):
```